.github/workflows/ci.yml

```diff
@@ -28,6 +25,8 @@ jobs:
         # the repo). Following the Principle of least privilege, we disable this as long
         # as we don't need it.
         persist-credentials: false
+    - name: Go golangci-lint and gotestsum
+      run: task install-deps
```

Comment:
Yes.
You can also omit completely the `name:` key, as I do in the GitHub Actions for `cogito`.

Reply:
Done as suggested.